src/main/resources/templates/numberquestion.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Number Question</title>
+</head>
+<body>
+<form>
+    <div th:fragment="num">
+        <div th:text="${question.getDescription()}"></div>
+        <input type="number" th:min="${question.getMinRange()}" th:max="${question.getMaxRange()}"/>
```

Comment:
Is this how number questions should look like? Maybe it should be a range?

Reply:
Yea just to add to this I think <input type="range"> will be good for our ui here

Reply:
Done! Changed to a range type of input!

Reply:
does the input show the min and max values